src/test/js/shared/custom_steps.js

```diff
@@ -14,8 +14,7 @@ module.exports = function() {
         this.fillField('password', '111');
         this.scrollPageToBottom();
         this.click('Sign in');
-        this.waitInUrl('/authorize', 60);
-        this.seeInField('#state', 'attemptone');
+        this.retry({retries:3, minTimeout:1000}).seeInCurrentUrl('state', 'attemptone');
```

Comment:
probably best to put the waitForText('Incorrect email...') before this line (and all the other ones)